ibis/pandas/execution/selection.py

```diff
@@ -76,7 +76,9 @@ def compute_projection_scalar_expr(expr, parent, data, scope=None, **kwargs):
 
 
 @compute_projection.register(ir.ColumnExpr, ops.Selection, pd.DataFrame)
-def compute_projection_column_expr(expr, parent, data, scope=None, **kwargs):
+def compute_projection_column_expr(
+    expr, parent, data, scope=None, timecontext=None, **kwargs
```

Comment:
Can you remove the default value for scope and timecontext here?

Reply:
done